.cargo/config.toml

```diff
@@ -0,0 +1,2 @@
+[build]
+target = "x86_64-unknown-linux-musl"
```

Comment:
As far as I'm aware there is no case we don't build with this target (locally or in CI)